### ISSUE TITLE
[Bug]: If you have a NVR and a Hub, the NVR itself isn't showing up

### ISSUE THREAD

**Issue by pvangorp** [bvis/aegis-hass#425]
### Describe the bug

If you have a NVR and a Hub, the Hub and the cameras are available in the integration. The NVR itself isn't available in the integration as a device. So you can't monitor the NVR itself. Disk space, temperature, cpu usage, ram usage, etc.

### Steps to reproduce

No additional steps required 

### Expected behavior

The NVR should be available as a device, just like in the Ajax app

### Home Assistant version

2026.8.2

### Integration version

1.17.0

### Installation method

HACS

### App label (co-branded app)

Ajax

### FCM push notifications configured?

No

### Relevant logs

```text

```

**Comment by pvangorp** [comment]
Also tested with FCM push notifications configured

**Comment by bvis** [comment]
Thanks — and the FCM datapoint is useful in a negative way: device modeling comes entirely from the device stream, push only routes activity, so "same with FCM on" confirms that axis is irrelevant here.

What decides where the fix lives is whether the NVR's own row arrives in that stream at all. Two possibilities: the stream does carry it and the integration is mishandling it (fixable in parsing — the integration already recognizes NVR-type rows), or the app reads NVR health (disk, CPU, RAM, temperature) from a separate endpoint the integration doesn't consume yet.

Could you attach the integration's diagnostics? **Settings → Devices & services → Aegis → three-dot menu → Download diagnostics.** It lists the device rows the stream actually delivered (ids and types only — it deliberately contains no personal names), which answers the question in one file.

**Comment by pvangorp** [comment]
Is this the part that causes the issue?

```
2026-08-21 17:30:30.290 DEBUG (MainThread) [custom_components.aegis_ajax.api.devices] Skipping unsupported device type: video_edge
2026-08-21 17:30:30.290 DEBUG (MainThread) [custom_components.aegis_ajax.api.devices] Skipping unsupported device type: video_edge
```

**Comment by bvis** [comment]
Those two lines were exactly it — thank you, that answered the question faster than the diagnostics would have. The recorder arrives as its own row type in the device stream (`video_edge`, distinct from the cameras' channel rows) and the integration was skipping it as unsupported.

Fixed in **`1.17.1-beta.3`**: the NVR now surfaces as a device with its name, online/offline state, and two new sensors — **channels online** and **channels total** — if your rows carry the counters.

To try it: HACS → Aegis → three-dot menu → Redownload → enable *show beta versions* → pick `1.17.1-beta.3` → restart Home Assistant.

Two things worth knowing when you test:

1. Since you saw TWO skipped rows, you may get two new device cards — if the second one isn't a recorder you recognize, say what it looks like.
2. **Disk space, CPU, RAM and temperature are deliberately not included yet.** They don't ride the row the integration receives — the Ajax app reads them from a separate per-device endpoint. Surfacing those is a different decision (it means adding a request the integration doesn't currently make) and I'd rather see first what your install actually delivers with this fix.

Whatever appears (or doesn't), a short note here calibrates the next step.

**Comment by pvangorp** [comment]
Thank you for your effort. I got 1 new device. I have 7 cameras connected.

<img width="1080" height="2340" alt="Image" src="https://github.com/user-attachments/assets/f944f951-c157-4382-9d77-d4e518d85a65" />

**Comment by bvis** [comment]
That screenshot settles it — exactly the expected result: the recorder card with **channels online 7 / total 7**, matching your 7 cameras, plus its tamper, connection and firmware state. And one card rather than two is correct: both of those skipped log lines were the same recorder, reported twice by the stream.

Thanks for the quick validation loop — the two log lines you spotted answered in a day what would otherwise have taken a diagnostics round-trip. The fix stays in the 1.17.1 beta line and this issue will close when that version ships as stable.

**Comment by pvangorp** [comment]
Will the other sensors also be added?

<img width="1080" height="2080" alt="Image" src="https://github.com/user-attachments/assets/440a10e3-dbc2-4c37-96db-eb5f176a3d0c" />

**Comment by bvis** [comment]
Good question — mostly no, and it's a deliberate line rather than an oversight. The states on the recorder card now (connection, tamper, channels online/total, firmware) all ride on data the integration already receives continuously. The internals on that app screen — CPU, RAM, hard-disk state and temperature, archive length, uptime — are different: the app fetches them on demand, only while you have that screen open. For the integration to show them as live sensors it would have to poll that endpoint on a schedule, and a hard rule of this project is to never generate more traffic against Ajax's servers than the official app does — that protects every user of the integration.

If any of those values ever turns out to be present in the data that already flows, it becomes free to expose and I'll gladly add it. Until then the recorder card stays as it is.

**Comment by pvangorp** [comment]
Thanks for your explanation. It is very clear

**Comment by bvis** [comment]
@pvangorp — **`1.17.1` is out as a stable release** with the recorder card in it, so the beta channel is no longer needed; a normal HACS update covers it. Your screenshot showing 7/7 channels on one card is what validated this, and the release notes say so. Closing as fixed — comment here if anything about the recorder card looks wrong on the stable build.

On the internal sensors (CPU, RAM, disk, uptime): still out, for the reason in my earlier comment — the app fetches those on demand while you have that screen open, and polling for them would mean more traffic against Ajax than the official app makes.
